packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -211,6 +211,11 @@ class CupertinoSlider extends StatefulWidget {
     properties.add(DoubleProperty('value', value));
     properties.add(DoubleProperty('min', min));
     properties.add(DoubleProperty('max', max));
+    properties.add(DiagnosticsProperty<ValueChanged<double>>('onChanged', onChanged));
+    properties.add(DiagnosticsProperty<ValueChanged<double>>('onChangeStart', onChangeStart));
+    properties.add(DiagnosticsProperty<ValueChanged<double>>('onChangeEnd', onChangeEnd));
```

Comment:
For any cases but onClick you want
    properties.add(ObjectFlagProperty<ValueChanged<double>>.has('onChangeStart', onChangeStart));

Reply:
Is this something we should try and codify in the fix?